services/contract_create.proto

```diff
@@ -177,4 +178,14 @@ message ContractCreateTransactionBody {
      * cover auto-renewal fees.
      */
     AccountID auto_renew_account_id = 15;
+
+    /**
+     * ID of the new account to which this contract is staking. If not staked to an account, the value is null. The default value is null.
+     */
+    AccountID staked_account_id = 17;
```

Comment:
I believe this should be a oneof with `int64 staked_node_id`. For updates, they can use the Int64Value wrapper. Same comment for other creates.

Reply:
Fixed. Thanks @steven-sheehy